ASCII_SUBSEQUENCES/Ascii_subsequences.java

```diff
@@ -0,0 +1,40 @@
+ /* Take as input str, a string. We are concerned with all the possible ascii-subsequences of str.
+    Also we have to print the count of such subsequences.
+    Sample input : ab
+    Sample output :
+     b 98 a ab a98 97 97b 9798
+     9 */
+
+import java.util.Scanner;
+
+public class Ascii_Subsequences {
+    public static void main(String args[]) {
+	Scanner obj = new Scanner(System.in);
```

Comment:
Indentation is not correct. There is a mix of 4 spaces and 1 tab in the whole file.

Reply:
Isnt 1 tab = 4 spaces?